extensionsGallery-insider.json

```diff
@@ -5036,30 +5036,30 @@
 					},
 					"versions": [
 						{
-							"version": "1.82.15",
-							"lastUpdated": "05/22/2023",
+							"version": "1.95.239",
+							"lastUpdated": "07/06/2023",
 							"assetUri": "",
 							"fallbackAssetUri": "fallbackAssetUri",
 							"files": [
 								{
 									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/copilot/1.82.15/GitHub.copilot-1.82.15.vsix"
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/copilot/1.95.239/GitHub.copilot-1.95.239.vsix"
 								},
 								{
 									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
 									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/copilot/Copilot-App-Icon.png"
 								},
 								{
 									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/copilot/1.82.15/README.md"
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/copilot/1.95.239/README.md"
 								},
 								{
 									"assetType": "Microsoft.VisualStudio.Services.Content.Changelog",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/copilot/1.82.15/CHANGELOG.md"
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/copilot/1.95.239/CHANGELOG.md"
 								},
 								{
 									"assetType": "Microsoft.VisualStudio.Code.Manifest",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/copilot/1.82.15/package.json"
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/copilot/1.95.239/package.json"
 								},
 								{
 									"assetType": "Microsoft.VisualStudio.Services.Content.License",
```

Comment:
the AzDataEngine dependency a few lines lower is currently `*`. Shouldn't it be the ADS version dependency start where copilot was first supported?

Reply:
@lewis-sanchez .did you build the vsix yourself?

Reply:
@kisantia the package.json in the visx doesn't include this property since it is not an ADS extension, having the vscode requirement is good enough.